updating migration guide (#286)
infinum/eightshift-docs: website/docs/migrations/9-10.md

- [eightshift/frontend-libs](https://github.com/infinum/eightshift-frontend-libs/releases/tag/8.6.2) - **8.6.2**
- [eightshift/libs](https://github.com/infinum/eightshift-libs/releases/tag/6.5.7) - **6.5.7**

## Optional changes

**Migration time: ~10min.**

> - In this version we changed the way we use header and footer blocks using reusable blocks. There is no need to update your project with this changes.

Reviewer: ```suggestion
- In this version we changed the way we use header and footer blocks using reusable blocks (synced patterns). There is no need to update your project with theese changes.
```

> - In this version we changed the way we use header and footer blocks using reusable blocks (synced patterns). There is no need to update your project with theese changes.

Reviewer 2: ```suggestion
- In this version we changed the way header and footer blocks are used. Synced patterns (formerly known as Reusable blocks) are now used. No changes in your project are needed.
```
